ci/build_python.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+if [[ -n "${DEBUG-}" ]]; then
+  set -x
+fi
+
+cd "$(dirname "$0")/../"
+
+./gradlew clean build preparePythonClasspath buildWheelDistribution --info --stacktrace
```

Comment:
doesn't `preparePythonClasspath` happen as part of `build`?

Reply:
Yes. However, if this changes in the future for whatever reason it will break this script without the changer noticing. I'd rather be explicit about what tasks are required and let gradle figure out what what to do.

Reply:
I don't think this makes a lot of sense. 

Firstly, because `build` always depends on `test`, and the python tests would fail if you hadn't done `preparePythonClasspath`. 

Secondly, because your order is weird. Gradle will just dumbly execute the tasks in order. So you're telling it first to do a full `build` (it will build every project*, including `python`) and then to do the `preparePythonClasspath` (again). 

So at the very least, I'd suggest swapping the order of `preparePythonClassPath` and `build`. But even so, I don't think it's necessary.

(*) P.S. Maybe as an optimisation you want `:keanu-python:build` instead of `build`? So that it doesn't waste time on unrelated tasks e.g. `:docs:build`.

Reply:
Yeah fair enough.

Oh really? I thought gradle would be smart enough to know that `preparePythonClasspath` had been done.

I'll change it to be `clean preparePythonClasspath buildWheelDistribution`.